pkgs/applications/networking/mumble/default.nix

```diff
@@ -60,8 +60,8 @@ let
         ] ++ (overrides.buildInputs or [ ]);
 
         cmakeFlags = [
-          "-D g15=OFF"
-          "-D CMAKE_CXX_STANDARD=17" # protobuf >22 requires C++ 17
+          (lib.cmakeBool "g15" false)
+          (lib.cmakeFeature "CMAKE_CXX_STANDARD" "17") # protobuf >22 requires C++ 17
```

Comment:
I don't see an advantage of using these "helpers" here. They only hide what's done. Please remove this patch.

Reply:
Under which circumstances would you see an advantage if using the helpers, @felixsinger?

Reply:
@felixsinger Sorry, I was under the impression that this was 'the orthodox way' in nixpkgs, I'll revert that.

Reply:
[x] done - @felixsinger do you need anything else to get this merged?